Cargo.toml

```
@@ -82,6 +82,7 @@ rand = "0.8.5"
 rand_chacha = "0.3.1"
 reqwest = "0.12.2"
 rocksdb = "0.22.0"
+rust-embed = { version = "8.5.0", features = ["mime-guess"] }
```

just the version should suffice

```suggestion
rust-embed = "8.5.0"
```

---

feature mime-guess is optional feature in rust-embed so this needs to be present otherwise it throws error that mimtype() is not found.

---

it's already accounted for in the `Cargo.toml` of `calimero-server`

---

I tried it but if I remove it even from root Cargo.toml it throws the error

---

`_` != `-`

https://github.com/calimero-network/core/blob/9cabb26747e98d9b194e12a6aa0c4a1e4f7d9758/Cargo.toml#L88

https://github.com/calimero-network/core/blob/9cabb26747e98d9b194e12a6aa0c4a1e4f7d9758/crates/server/Cargo.toml#L31